README.md

```diff
@@ -9,26 +9,42 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="200" height="235" align="right" hspace="12" />
+
+Ramda makes it simple for you to build complex logic through
+functional composition. This means sticking simple functions together.
+It is designed to help you get you job more elegantly with simpler code.
+Ramda does not mutate your state. Any data you throw at Ramda is safe from
+hidden state changes, and this is a very powerful weapon to battle complexity.
```

Comment:
perhaps a list here, e.g.:

> Because:
    * Ramda makes it simple to build complex logic through functional composition, i.e. sticking simple functions together.
    * Ramda functions are virtually all curried by default.
    * Ramda *does not mutate*. We do our best to make sure *none* of the functions in Ramda mutate their inputs.
    * Ramda can help you get you job more elegantly with simpler code.